Build failure with GHC 8

```
> /tmp/stackage-build8$ stack unpack QuasiText-0.1.2.5
Unpacked QuasiText-0.1.2.5 to /tmp/stackage-build8/QuasiText-0.1.2.5/
> /tmp/stackage-build8/QuasiText-0.1.2.5$ runghc -clear-package-db -global-package-db -package-db=/home/stackage/work/builds/nightly/pkgdb Setup configure --package-db=clear --package-db=global --package-db=/home/stackage/work/builds/nightly/pkgdb --libdir=/home/stackage/work/builds/nightly/lib --bindir=/home/stackage/work/builds/nightly/bin --datadir=/home/stackage/work/builds/nightly/share --libexecdir=/home/stackage/work/builds/nightly/libexec --sysconfdir=/home/stackage/work/builds/nightly/etc --docdir=/home/stackage/work/builds/nightly/doc/QuasiText-0.1.2.5 --htmldir=/home/stackage/work/builds/nightly/doc/QuasiText-0.1.2.5 --haddockdir=/home/stackage/work/builds/nightly/doc/QuasiText-0.1.2.5 --flags=
Configuring QuasiText-0.1.2.5...
> /tmp/stackage-build8/QuasiText-0.1.2.5$ runghc -clear-package-db -global-package-db -package-db=/home/stackage/work/builds/nightly/pkgdb Setup build
Building QuasiText-0.1.2.5...
Preprocessing library QuasiText-0.1.2.5...
[1 of 1] Compiling Text.QuasiText   ( src/Text/QuasiText.hs, dist/build/Text/QuasiText.o )

src/Text/QuasiText.hs:19:10: error:
    Duplicate instance declarations:
      instance [incoherent] Lift Text
        -- Defined at src/Text/QuasiText.hs:19:10
      instance Lift Text
        -- Defined in ‘th-lift-instances-0.1.7:Instances.TH.Lift’
```

--- a/QuasiText.cabal
+++ b/QuasiText.cabal
@@ -16,5 +16,5 @@ cabal-version:       >=1.8
 
 library
   exposed-modules:     Text.QuasiText
-  build-depends:       base >= 4.5.0.0 && < 5.0.0.0, template-haskell, haskell-src-meta, attoparsec, text
+  build-depends:       base >= 4.5.0.0 && < 5.0.0.0, template-haskell, haskell-src-meta, attoparsec, text, th-lift-instances
   hs-source-dirs:      src

--- a/src/Text/QuasiText.hs
+++ b/src/Text/QuasiText.hs
@@ -3,6 +3,7 @@
 -- A simple 'QuasiQuoter' for 'Text' strings. Note that to use 'embed' you need to use the OverloadedStrings extension.
 
 module Text.QuasiText (embed, Chunk (..), getChunks) where
+import Instances.TH.Lift () -- for the `instance Lift Text`
 import Language.Haskell.TH.Quote
 import Language.Haskell.TH.Syntax
 import Language.Haskell.TH
@@ -15,9 +16,6 @@ import Data.Char
 
 import Data.Monoid
 import Control.Applicative
-                        
-instance Lift Text where
-    lift = litE . stringL . T.unpack
 
 data Chunk 
     = T Text -- ^ text
@@ -57,7 +55,7 @@ embed = QuasiQuoter
                             Left  e -> error e
                             Right e -> appE [| toText |] (return e)
 
-                        V t -> appE [| toText |] (global (mkName (T.unpack t)))
+                        V t -> appE [| toText |] (varE (mkName (T.unpack t)))
 
         in appE [| T.concat |] (listE chunks)
     }

